tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -33,7 +33,7 @@
     environment: "{{ he_cmd_lang }}"
     changed_when: true
     register: ovf_store_status
-    retries: 12
+    retries: 120
```

Comment:
maybe we can change it to:
    retries: 60
    delay: 20
to reduce the console output each 10 seconds

Reply:
I don't mind trying to get this patch in, but it seems like it should not be needed - see also:

https://gerrit.ovirt.org/110324

https://lists.ovirt.org/archives/list/infra@ovirt.org/thread/Q7QAW24L5AYRJVBFYWCTE7UW4DSFTSFN/

Not sure latter was already handled, but: 1. Latest appliance still did not include fixed wildfly 2. Fixed wildfly now does appear on resources.ovirt.org. 3. I am now rebuilding the appliance. If it will include the fixed wildfly, I'll try the he-basic suite. If it passes, I'll consider the issue fixed. If you then still want me to handle current patch, ok :-)

Reply:
Great!! thanks for the detailed answer!!
I'll update if I'll see it again with the new appliance

Reply:
eventually fixed by: https://gerrit.ovirt.org/110397 . I am closing the PR.
If anyone later feels like reopening, please consider analyzing the status (checking engine.log, perhaps other stuff) and report whatever you find - just extending the wait should not be needed, and if it fails eventually, it's just somewhat more annoying.

I still do not know this code well, but AFAICT from reading parts of it, it's designed to do things relatively quickly - including e.g. a task "Set OVF update interval to 1 minute", etc.